src/app/recent/nearby/map-nearby.tsx

```diff
@@ -26,6 +28,20 @@ function BirdCard({
   isSelected: boolean;
 }) {
   const cardRef = useRef<HTMLDivElement>(null);
+  const [birdImage, setBirdImage] = useState<any>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    const fetchBirdImage = async () => {
+      const image = await getBirdImage(sighting.sciName);
+      setBirdImage(image);
+      setIsLoading(false);
+    };
+
+    fetchBirdImage();
+  }, [sighting.sciName]);
```

Comment:
Future PR:

If we're fetching data on the client is it worth investing in adding something like [Tanstack Query](https://tanstack.com/query/latest/docs/framework/react/overview)? 

It would solve the error case you mentioned earlier and mean we don't have to manually manage state, it would also give us a nice cache so that we're not over-fetching images when navigating around the map.